src/Dashboard/docs/New-AzGrafana.md

```diff
@@ -0,0 +1,335 @@
+---
+external help file:
+Module Name: Az.Dashboard
+online version: https://docs.microsoft.com/powershell/module/az.dashboard/new-azgrafana
+schema: 2.0.0
+---
+
+# New-AzGrafana
+
+## SYNOPSIS
+Create or update a workspace for Grafana resource.
+This API is idempotent, so user can either create a new grafana or update an existing grafana.
+
+## SYNTAX
+
+```
+New-AzGrafana -Name <String> -ResourceGroupName <String> [-SubscriptionId <String>] [-ApiKey <ApiKey>]
+ [-AutoGeneratedDomainNameLabelScope <AutoGeneratedDomainNameLabelScope>]
+ [-DeterministicOutboundIP <DeterministicOutboundIP>] [-IdentityType <ManagedServiceIdentityType>]
+ [-IdentityUserAssignedIdentity <Hashtable>] [-Location <String>] [-PublicNetworkAccess <PublicNetworkAccess>]
```

Comment:
Please update Location as required.

Reply:
Swagger Link : https://github.com/Azure/azure-rest-api-specs/blob/main/specification/dashboard/resource-manager/Microsoft.Dashboard/stable/2022-08-01/grafana.json#L719
The swagger format was not "required":true

Reply:
I remember you said it's required in your manual test?

Reply:
Yes, but in fact, in the example I provided, all of the parameters used are required.
I am now ready to mark these parameters "Mandatory" under the Custom folder

Reply:
I can't see it's mandatory in the syntax of New-AzGrafana.md

Reply:
Update some parameters and mark them as required